### DISCUSSION TITLE
Documentation about self-host is unclear

### DISCUSSION THREAD

**Discussion by drHuangMHT** [rustdesk/rustdesk#808] [General]
The documentation only shows how to get the server and relay up and running and few client-side instructions. As a newcomer I cannot connect to the host successfully. I started clients on both computer(Debian 11 with Plasma and Windows 11), set the ID server address and gave the ID but no luck. Maybe there's room for improvement on the documentation?
----Update----
I know it's kind of a dumb idea to use desktop environment when you have another Windows computer. Yes, I want to control a Linux machine on Windows. I could have used ssh but the issue is here, I can't get things working just by looking at the documentation.
----Update----
I can control my Windows machine now(through relay hosted by myself, terminal showed the handshake) but still a no-go the other way. On the Windows side it says unable to connect to relay but the status on the bottom says ready. If I randomly give an ID it will tell me ID does not exist. I'm confused.

**Comment by xperus** [comment]
Same here I have set up an off site relay and ID sever (forcing) a key and can see the other machines on my local network in the apps/client discovered list but when I try to connect I get the the same IS DOES NOT EXIST error message. I have a mix of win10 Windows server Debian and Ubuntu machines.

**Comment by rustdesk** [comment]
This video should be helpful. https://www.youtube.com/watch?v=9nzHm3xGz2I

Another one in French: https://www.youtube.com/watch?v=J7t32K5--cM

BTW, the document is open source also, https://github.com/rustdesk/doc.rustdesk.com

**Reply by drHuangMHT** [reply]
The video shows how to set up rustdesk server using docker compose, which is different from my case. Also the in video the server is hosted on cloud but in my case I only need local access. The server is hosted on the same machine that I want to have access to so maybe this will cause some conflicts? What if I don't have a third machine to host the server? I'll consider changing the port for now and see what happens.

**Reply by drHuangMHT** [reply]
Changing ports doesn't help. Strangely enough the host name(user@host, the thing that shows up to the left side of the prompt in terminal, I don't know what it's called) got displayed on my Windows machine. Maybe this is a bug? Also the video does not demonstrate how to control a Linux machine on Windows.
